### PR TITLE
ALPHA-283-add-collection-meta

### PR DESCRIPTION
**Pull request by augur**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -3406,6 +3408,64 @@ components:
...
+          description: "Name of the collection"
+        description:
+          type: string
+          description: "Description of the collection"
+        image:
+          "$ref": "#/components/schemas/Media"
+        external_link:
+          type: string
+          description: "External link to the original website for the collection"
+        seller_fee_basis_points:
```
[the remark this answers is not in the document]
Comment (augur) [reply openapi.yaml]: Fixed

File: openapi.yaml [hunk 2]
```diff
@@ -3406,6 +3408,64 @@ components:
...
+          type: integer
+          format: int32
+          description: "Indicates a 1% seller fee"
+        fee_recipient:
+          description: "Where seller fees will be paid to"
+          "$ref": "#/components/schemas/UnionAddress"
+
+    Media:
```
[the remark this answers is not in the document]
Comment (augur) [reply openapi.yaml]: Switched to `MetaContent`

File: openapi.yaml [hunk 3]
```diff
@@ -3406,6 +3408,33 @@ components:
...
+        - name
+      type: object
+      properties:
+        name:
+          type: string
+          description: "Name of the collection"
+        description:
+          type: string
+          description: "Description of the collection"
+        content:
```
Comment (yakischik) [inline openapi.yaml:3422]: I think we can do it required, but always send empty array

Comment (augur) [reply openapi.yaml:3423]: Agree, fixed

**Pull request merged**